cfme/tests/automate/custom_button/test_infra_objects.py

```diff
@@ -163,8 +163,13 @@ def test_custom_button_automate(appliance, request, submit, setup_obj, button_gr
         # Entity count depends on the destination for `All` available entities and
         # `Details` means a single entity.
         if destination in ["All", "VMsOnly", "TemplatesOnly"]:
-            entity_count = len(view.entities.entity_names)
-            view.paginator.check_all()
+            try:
```

Comment:
Consider coming back and modifying the views so that paginator is always within entities, if possible.

I realize sometimes with the configure pages, paginator isn't really part of entities, or there isn't an entities nested view anyway.